packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
while `on` events aren't implemented yet

Reply:
Please add this comment in the code so we know when to go remove that.